cesium/internal/version/version.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2023 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+package version
+
+type Version = uint8
+
+const CurrentVersion = 2
```

Comment:
Just call this `Current` and make sure its of type `Version` because I think it will be an int right now